cluster/gce/manifests/glbc.manifest

```diff
@@ -17,7 +17,7 @@ spec:
   terminationGracePeriodSeconds: 600
   hostNetwork: true
   containers:
-  - image: k8s.gcr.io/ingress-gce-glbc-amd64:v1.6.1
+  - image: gcr.io/k8s-ingress-image-push/ingress-gce-glbc-amd64:v1.11.0
```

Comment:
the registry change is surprising... is this a staging image or a production image?

Reply:
This is the registry that prow puts all the images it creates: https://github.com/kubernetes/test-infra/blob/a92a9a94ced9d3788ac4423ddbdc2f75d7088bda/config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml#L108-L130

Reply:
do we want to reference a CI-populated registry here, or an "official releases only" registry?

Reply:
I am not too familiar with the `k8s.gcr.io` registry, but I do not see the image in `https://github.com/kubernetes/k8s.io`. I believe the previous image was working because it was created before the container registry migration.

Tags on the `ingress-gce` repo are how releases are cut, so the version tagged images are equivalent to our releases.

@freehan are there other registries that we maintain?